python/lsst/pipe/analysis/colorAnalysis.py

```diff
@@ -373,11 +373,28 @@ def plotStarColorColor(self, catalogs, filenamer, dataId, butler=None, camera=No
         filters = set(catalogs.keys())
         color = lambda c1, c2: (mags[c1] - mags[c2])[good]
         if filters.issuperset(set(("HSC-G", "HSC-R", "HSC-I"))):
+            # Do a linear fit to regions defined in Ivezic transforms
```

Comment:
I don't think this is worth doing. Not only are the ranges not appropriate for the HSC photometric system (as you've said), but the locus isn't even linear. That's the whole reason `colorColorPolyFitPlot` exists in the first place.